src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -427,6 +428,18 @@ public bool HasUnsavedChanges
         }
 
         /// <summary>
+        /// Returns if current workspace is readonly.
+        /// </summary>
+        public bool IsReadOnly
```

Comment:
just curious, any reason to create the backing field vs just using auto getter/setter syntax